spec/airbrake_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+RSpec.describe Airbrake do
+  describe "#add_rack_builder" do
+    it "adds new builder to the chain" do
+      builder = proc { |_, _| nil }
+      expect(Airbrake::Rack::NoticeBuilder).to receive(:add_builder) do |_, &block|
```

Comment:
Sure, understandable, that said, you could cleanup after you've done checking. It's important to test this properly because it's public API.